src/sentry/api/endpoints/organization_config_integrations.py

```diff
@@ -15,28 +15,30 @@ def get(self, request, organization):
         has_catchall = features.has('organizations:internal-catchall',
                                     organization,
                                     actor=request.user)
+        has_github_apps = features.has('organizations:github-apps',
+                                       organization,
+                                       actor=request.user)
 
         for provider in integrations.all():
             metadata = provider.metadata
             metadata = metadata and metadata._asdict() or None
-            if not has_catchall and provider.key in settings.SENTRY_INTERNAL_INTEGRATIONS:
-                continue
-            providers.append(
-                {
-                    'key': provider.key,
-                    'name': provider.name,
-                    'metadata': metadata,
-                    'canAdd': provider.can_add,
-                    'canAddProject': provider.can_add_project,
-                    'setupDialog': dict(
-                        url='/organizations/{}/integrations/{}/setup/'.format(
-                            organization.slug,
-                            provider.key,
-                        ),
-                        **provider.setup_dialog_config
-                    )
-                }
-            )
+            if has_github_apps or has_catchall and provider.key in settings.SENTRY_INTERNAL_INTEGRATIONS:
```

Comment:
actually this doesn't work...I don't think. @macqueen not sure the best way to make sure _both_ slack and github make it through

Reply:
hmm yeah lol this is complicated. i think something like this would work?

```python
internal_integrations = {i for i in settings.SENTRY_INTERNAL_INTEGRATIONS if i != 'github' or not has_github_apps}
if not has_catchall and provider.key in internal_integrations:
    continue
providers.append(
...
```

Reply:
kewl, yeah that works